access-modifier-checker/src/main/java/org/kohsuke/accmod/impl/Checker.java

```diff
@@ -228,122 +236,7 @@ public void checkClass(File clazz) throws IOException {
         FileInputStream in = new FileInputStream(clazz);
         try {
             ClassReader cr = new ClassReader(in);
-            cr.accept(new ClassVisitor(Opcodes.ASM5) {
```

Comment:
easier to review the [pre-refactored diff](https://github.com/kohsuke/access-modifier/pull/16/commits/d55f1e1ecc51c767284d1a73ef06c887f981bb19)